<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock $(env HOME)/rosbag/infant/dkan_outdoor/2019-01-13-15-46-58.bag -s 60"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find pc_transform)/rviz_config/pc_tf_transform_velodyne.rviz"/>

	<!-- sensor driver -->
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

	<!-- tf -->
	<node pkg="tf" type="static_transform_publisher" name="static_transform_publisher" args="5 0 0 0 0 0 base_link velodyne 100"/>

	<!-- main -->
	<node pkg="pc_transform" type="tf_transform" name="tf_transform" output="screen">
		<remap from="/point_cloud" to="/velodyne_points"/>
		<remap from="/point_cloud/transformed" to="/velodyne_points/transformed"/>
		<param name="target_frame" type="string" value="base_link"/>
	</node>
</launch>
